### ISSUE TITLE
GET on my Glitch API doesn't work

### ISSUE THREAD

**Issue by aashutoshrathi** [liyasthomas/postwoman#16]
**Describe the bug**
A simple GET request on http://aashutoshrathi.glitch.me/api/self/gh gives the error

**To Reproduce**
1. Go to  https://liyasthomas.github.io/postwoman/
2. Select Method as "GET"
3. URL as `http://aashutoshrathi.glitch.me`
4. And Path as `/api/self/gh`
5. Then send a request.

**Expected behavior**

It should result in response same as http://aashutoshrathi.glitch.me/api/self/gh

**Screenshots**

![image](https://user-images.githubusercontent.com/21199234/63550738-ee71c600-c550-11e9-818b-d4a94c6da078.png)

**Comment by striebwj** [comment]
Use https:// instead of http://. 

Here is a screenshot from when I ran it using https://:
![Screen Shot 2019-08-22 at 5 51 22 PM](https://user-images.githubusercontent.com/14165147/63552392-877aef80-c505-11e9-8cbb-139cd32e75f9.png)

Here is a comment about http not working right now: [link to comment](https://github.com/liyasthomas/postwoman/issues/2#issuecomment-523870224)

**Comment by liyasthomas** [comment]
Possible duplicate of #2.
CORS is a server side security policy feature which blocks API calls from any non-HTTPS requests.

**Comment by aashutoshrathi** [comment]
I think I should close it then.

**Comment by jamesscottbrown** [comment]
This is not a [CORS](https://developer.mozilla.org/en-US/docs/Glossary/CORS) issue, but rather a [Mixed Content](https://developer.mozilla.org/en-US/docs/Web/Security/Mixed_content) issue. An `XMLHttpRequest ` or `fetch()` request is considered "active content"; "mixed active content" such as a `XMLHttpRequest ` or `fetch()` to an `http://` URL from an `https://` URL has been blocked by default since Firefox 23 and is also blocked by Chrome.

In this case, both URLs provide the same CORS headers:
```
< access-control-allow-origin: *
< access-control-allow-headers: Origin, X-Requested-With, Content-Type, Accept
```

Also, CORS does not block cross-origin requests:  the *same-origin policy* prevents them by default, and *CORS* headers can be used to opt-in to allowing them.
